src/core/providers/ogr/qgsogrprovider.cpp

```diff
@@ -35,7 +35,6 @@ email                : sherman at mrcc.com
 #include "qgsembeddedsymbolrenderer.h"
 #include "qgsprovidersublayerdetails.h"
 #include "qgsvectorlayer.h"
-#include "qgsproviderregistry.h"
```

Comment:
Was it removed by mistake?

Reply:
yes and no: I trusted my editor (which uses clang-analyzer) that said it wasn't used, but I see my editor was wrong. I'll put it back.